plugins/modules/website.yml

```diff
@@ -64,16 +181,26 @@ EXAMPLES: |
       physical_path: C:\inetpub\wwwroot
       state: started
 
-  - name: Create a website, set all values and start it
+  - name: Create a website with two bindings, set all values and starts it
```

Comment:
We should look at adding a few more examples for things like

+ `set: []` to remove all bindings
+ Some with add/remove to show how they can be specified.